eras/shelley/impl/testlib/Test/Cardano/Ledger/Shelley/ImpTest.hs

```diff
@@ -302,6 +304,7 @@ instance ShelleyEraImp era => ImpSpec (LedgerSpec era) where
             ImpTestEnv
               { iteFixup = fixupTx
               , iteCborRoundTripFailures = True
+              , iteHook = \_ _ _ _ -> pure ()
```

Comment:
I think we should be more specific about where this hook gets called, so a better name would be something like `iteSubmitTxHook`

Reply:
Totally agree with the suggestion of renaming. I would even go further and suggest avoiding using name `Hook`. We only used it in passing to describe this concept. A better name would be: `iteExpectLedgerRuleConformance`